ui/src/components/QSharingIcon.vue

```diff
@@ -21,6 +21,22 @@ export default {
     }
   },
   methods: {
+    encodeHashTags () {
+      let tags
+
+      if (this.key === 'facebook' && this.hashtags?.length > 0) {
+        this.hashtags.forEach(tag => {
+          if (!tags) tags = tag
+          else tags += '%23' + tag
+        })
+      }
+
+      if (this.key === 'twitter' && this.hashtags?.length > 0) {
+        tags = this.hashtags.join(',')
+      }
+
+      return tags
+    },
```

Comment:
Please make sure you test your code.
- `this.key` does not exist. Did you mean `this.type`?
- due to how facebook tag condition is written, it generates  a tag like this: `social%23share` which I believe is not 

My suggestion
```suggestion
    encodeHashTags () {
 
      // facebook - accept only one hash 
      if (this.key === 'facebook' && this.hashtags?.length > 0) {
        return '%23' +t his.hashtags[0]
      } else if (this.key === 'twitter' && this.hashtags?.length > 0) {
        return this.hashtags.join(',')
      }
      return 
    },
```